lambdas/es/indexer/document_queue.py

```diff
@@ -31,33 +35,15 @@
 # signifies that the object is truly deleted, not to be confused with
 # s3:ObjectRemoved:DeleteMarkerCreated, which we may see in versioned buckets
 # see https://docs.aws.amazon.com/AmazonS3/latest/dev/NotificationHowTo.html
-OBJECT_DELETE = "ObjectRemoved:Delete"
-OBJECT_PUT = "ObjectCreated:Put"
-RETRY_429 = 5
 QUEUE_LIMIT_BYTES = 100_000_000  # 100MB
+RETRY_429 = 5
 
 
-def transform_meta(meta):
-    """ Reshapes metadata for indexing in ES """
-    helium = meta.get("helium", {})
-    user_meta = helium.pop("user_meta", {}) or {}
-    comment = helium.pop("comment", "") or ""
-    target = helium.pop("target", "") or ""
-
-    meta_text_parts = [comment, target]
-
-    if helium:
-        meta_text_parts.append(json.dumps(helium))
-    if user_meta:
-        meta_text_parts.append(json.dumps(user_meta))
-
-    return {
-        "system_meta": helium,
-        "user_meta": user_meta,
-        "comment": comment,
-        "target": target,
```

Comment:
Is this no longer used by the serialization/deserialization codepaths?

Reply:
Not sure actually. But we should rely on manifests and only manifests for metadata. The serdes code uses file extensions, I believe.